SymSettings.cfg

```diff
@@ -1,6 +1,31 @@
 ### MAIN ###
```

Comment:
The easiest way to fix the merge conflict in this file is probably to make the rest of the merges, compile symbulation, and regenerate this settings file from scratch